app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -39,9 +39,10 @@ function getQuoteTitle(element) {
 }
 
 function fixQuotes(str) {
-  // u+201c “
-  // u+201d ”
-  return str.replace(/[\u201C\u201D]/g, '"');
+  // u+201c, u+201d = “ ”
+  // u+2018, u+2019 = ‘ ’
+  let new_str = str.replace(/[\u201C\u201D]/g, '"');
+  return new_str.replace(/[\u2018\u2019]/g, "'");
```

Comment:
This could just be condensed to `return str.replace(/[\u201C\u201D]/g, '"').replace(/[\u2018\u2019]/g, "'")`, right?